images/python-aws-bash/Dockerfile

```diff
@@ -1,10 +1,8 @@
 FROM python:3
 
-# bug in spack s3 handling requires older botocore:
-#     https://github.com/spack/spack/issues/28830
 RUN pip install --upgrade \
-    awscli==1.22.46 \
-    boto3==1.20.35 \
-    botocore==1.23.46
+    awscli \
+    boto3 \
+    botocore
```

Comment:
Could these be moved to a `requirements.txt` file like we have for the rest of our custom docker images?

Reply:
Yes definitely, I don't know why I didn't do that originally.